docs/data/datamodel.md

```diff
@@ -1,8 +1,333 @@
+
+
+#### Person
+
+| Field            | Type     | Cardinality | Restrictions                           |
+| ---------------- | -------- | ----------- | -------------------------------------- |
+| `__id`           | string   | 1           |                                        |
+| `__type`         | string   | 1           | Literal 'Person'                       |
+| `givenNames`     | string[] | 1-n         |                                        |
+| `familyNames`    | string[] | 1-n         |                                        |
+| `jobTitles`      | string[] | 0-n         |                                        |
+| `affiliations`   | id[]     | 0-n         | Organization IDs                       |
+| `address`        | address  | 0-1         |                                        |
+| `email`          | string   | 0-1         |                                        |
+| `secondaryEmail` | string   | 0-1         |                                        |
+| `authorityRefs`  | url[]    | 0-n         | References to external authority files |
+
+
+#### Organization
+
+| Field             | Type        | Cardinality | Restrictions                           |
+| ----------------- | ----------- | ----------- | -------------------------------------- |
+| `__id`            | string      | 1           |                                        |
+| `__type`          | string      | 1           | Literal 'Organization'                 |
+| `name`            | string      | 1           |                                        |
+| `url`             | url         | 1           |                                        |
+| `address`         | address     | 0-1         |                                        |
+| `email`           | string      | 0-1         |                                        |
+| `alternativeName` | lang_string | 0-1         |                                        |
+| `authorityRefs`   | url[]       | 0-n         | References to external authority files |
+
+
+#### Grant
+
+| Field     | Type   | Cardinality | Restrictions               |
+| --------- | ------ | ----------- | -------------------------- |
+| `__id`    | string | 1           |                            |
+| `__type`  | string | 1           | Literal 'Grant'            |
+| `funders` | id[]   | 1-n         | Person or Organization IDs |
+| `number`  | string | 0-1         |                            |
+| `name`    | string | 0-1         |                            |
+| `url`     | url    | 0-1         |                            |
+
+
+### Value Types
+
+#### lang_string
+
+Object with a language tag as key and a string as value.
+
+```json
+{
+    "en": "Lorem ipsum in English.",
+    "de": "Lorem ipsum auf Deutsch."
+}
+```
+
+
+#### date
+
+String with the format `YYYY-MM-DD`.
+
+
+#### url
+
+An object representing a URL. 
+Depending on the `type` field,
+the URL may be a generic URL
+or a more specific link, like P PID
```

Comment:
```suggestion
or a more specific link, like PID
```

Reply:
meant to write "like a PID", thanks for spotting.